chucks-pick3-web/angular-app/src/app/shared/components/app.component/app.component.ts

```diff
@@ -9,18 +10,27 @@ import {TranslateService} from '@ngx-translate/core';
 })
 
 export class AppComponent {
+  title = 'angular-app';
+  public showSplashPage = true;
+
   constructor(public logInService: LogInValidationService,
-              private translate: TranslateService) {
+              private translate: TranslateService,
+              public browserLocaleService: BrowserLocaleService) {
     translate.setDefaultLang('en');
+    this.setLanguage();
   }
 
-  title = 'angular-app';
-  public showSplashPage = true;
 
-  switchLanguage(es: string) {
-    this.translate.use(es);
+  public setLanguage() {
```

Comment:
Renamed method to `useBrowserLanguage()`.

Reply:
Renamed method to useBrowserLanguage().